### PR TITLE
feat: adding a field in the config that telle where to send an http p…

### LINKED ISSUE #18: bonus: more advance logging
request http to a push bullet notifier equivalent.

### PR DESCRIPTION
**Pull request by a-boring-man**
…ost message in case of a process not launchin correctly, we can of course easly extend the range of event that involved a request

### CODE REVIEW

File: src/logs/main.rs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+// use to demonstrate that with request it work
+// use reqwest;
+// use serde_json::json;
+
+// #[tokio::main]
+// async fn main() -> Result<(), Box<dyn std::error::Error>> {
+//     let access_token = "o.8bTjZDwKe70TfNqzjMm42HfCcSYkFqA5";
+//     let client = reqwest::Client::new();
+
```
Comment (tdameros) [inline src/logs/main.rs]: Can you use an .env file to store secret tokens ?

File: src/server/server.rs [hunk 2]
```diff
@@ -24,6 +27,16 @@ pub mod process_manager;
...
+    #[cfg(feature = "reqwest")]
+    let access_token = "o.8bTjZDwKe70TfNqzjMm42HfCcSYkFqA5".to_string();
+    #[cfg(feature = "reqwest")]
+    let title = "Test Notification".to_string();
+    #[cfg(feature = "reqwest")]
+    let body = "This is a test notification from Rust!".to_string();
+
+    #[cfg(feature = "reqwest")]
+    send_notification(access_token, title, body);
+
```
Comment (tdameros) [inline src/server/server.rs]: Can you use an .env file to store secret tokens ?

And maybe remove this test notification ?

**Review by tdameros — CHANGES_REQUESTED** [review]

File: src/server/server.rs (hunk 2 above)
Comment (a-boring-man) [reply src/server/server.rs]: it was to show that a simple request didn't work, i removed it

File: src/logs/main.rs (hunk 1 above)
Comment (a-boring-man) [reply src/logs/main.rs]: issue taken cared of

**Review by tdameros — APPROVED** [review]

**Pull request merged**